plugin/restore/mysql/mysql.go

```diff
@@ -50,9 +54,64 @@ func New(driver *mysql.Driver, instance *mysqlutil.Instance, connCfg db.Connecti
 	}
 }
 
-// replayBinlog restores the database using incremental backup in time range of [config.Start, config.End).
-func replayBinlog(ctx context.Context, config api.BinlogInfo, binlogDir string) error {
-	return fmt.Errorf("Unimplemented")
+// ReplayBinlog replays the binlog about `originDatabase` from `startBinlogInfo.Position` to `targetTs`.
+func (r *Restore) replayBinlog(ctx context.Context, originalDatabase, pitrDatabase, binlogDir string, startBinlogInfo api.BinlogInfo, targetTs int64) error {
+	if err := r.SyncLatestBinlog(ctx, binlogDir); err != nil {
```

Comment:
Can we make this return the directory.

The current style is less readable as we don't know the exact info SyncLatestBinlog prepares, which enables getReplayBinlogPathList to run properly.

It's always desirable to pass dependency explicitly instead of relying on the internal side effect

Reply:
> Can we make this return the directory

IMO it's `binlogDir`.

Reply:
This is my fault. We don't need `binlogDir` as an argument now, we save it in `Restore` struct.